vyxal/helpers.py

```diff
@@ -0,0 +1,111 @@
+"""
+File: helpers.py
+Description: This is where the cool functions go that help out stuff.
+They aren't directly attached to an element. Consequently, you need to
+use type annotations here.
+"""
+
+import textwrap
+from typing import Union
+
+from vyxal import lexer
+
+
+def indent_str(string: str, indent: int, end="\n") -> str:
+    r"""
+    Indent a multiline string with 4 spaces.
+
+    Parameters
+    ----------
+    string: str
+        The string to be indented.
+    indent: int
+        How many levels of indentation are required.
+
+    Returns
+    -------
+    str
+        `string`, indented by `(4 * indent)` spaces, after `\n`.
```

Comment:
what does "after `\n`" mean here?

Reply:
I don't know. Ask @ysthakur

Reply:
But it seems it means that it only inserts indents after newlines: https://docs.python.org/3/library/textwrap.html#textwrap.indent